mica/report/tests/test_write_report.py

```diff
@@ -9,16 +9,21 @@
 
 from .. import report
 
+user = os.environ.get('USER') or os.environ.get('LOGNAME')
+
 try:
     import Ska.DBI
-    user = os.environ.get('USER') or os.environ.get('LOGNAME')
     with Ska.DBI.DBI(server='sqlsao', dbi='sybase', user=user, database='axafvv') as db:
         HAS_SYBASE_ACCESS = True
 except Exception:
-    if on_head_network() and not has_sybase():
-        warn("On HEAD but no sybase access. Run ska_envs or define SYBASE/SYBASE_OCS")
     HAS_SYBASE_ACCESS = False
 
+    # If the user should have access, warn about the issue.
+    if (on_head_network() and not has_sybase() and
+            os.path.exists(os.path.join(os.environ['SKA'], 'data', 'aspect_authorization',
```

Comment:
You should just start using `Path` at every opportunity. It is very much nicer in almost every case:
`Path(os.environ['SKA'], 'data', 'aspect_authorization', f'sqlsao-axafvv-{user}').exists()`.